t/cobrand/isleofwight.t

```diff
@@ -127,7 +133,94 @@ subtest "fixing passes along the correct message" => sub {
         is $id, 248, 'correct update ID returned';
         $cgi = CGI::Simple->new($o->test_req_used->content);
         like $cgi->param('description'), qr/^\[The customer indicated that this issue had been fixed/, 'Fixed message included';
+        $p->comments->delete;
+        $p->delete;
+    };
+};
+
+my ($p) = $mech->create_problems_for_body(1, $isleofwight->id, '', { bodies_str => $isleofwight->id, user => $user });
+$p->update( { cobrand => 'isleofwight' } );
```

Comment:
Tiny thing, but you can include this in the line above, and you don't need to provide bodies_str (or the user, unless I missed where that was used?), what you provide overrides the defaults.